### PR TITLE
KTL-138 Replace broken links in Docs

### PR DESCRIPTION
**Pull request by nikpachoo**
Hey team,
Several outdated links in the Docs have been found.
For some of them I found alternatives, but please review the suggested ones.

### CODE REVIEW

File: docs/topics/ksp/ksp-why-ksp.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ is fairly quick. Almost all of the build overhead resides in the stub generation
 reduce the time spent in the compiler by 25%.
 
 For performance evaluation, we implemented a [simplified version](https://github.com/google/ksp/releases/download/1.4.10-dev-experimental-20200924/miniGlide.zip)
-of [Glide](https://github.com/bumptech/glide) in KSP to make it generate code for the [Tachiyomi](https://github.com/inorichi/tachiyomi) project.
+of [Glide](https://github.com/bumptech/glide) in KSP to make it generate code for the Tachiyomi project.
```
Comment (nikpachoo) [inline docs/topics/ksp/ksp-why-ksp.md:61]: I haven't found an actual link.

File: docs/topics/kotlin-osgi.md [hunk 2]
```diff
@@ -94,7 +94,7 @@ dependencies {
 ### Why not just add required manifest options to all Kotlin libraries
 
 Even though it is the most preferred way to provide OSGi support, unfortunately it couldn't be done for now due to so called
-["package split" issue](http://wiki.osgi.org/wiki/Split_Packages) that couldn't be easily eliminated and such a big change is
```
Comment (nikpachoo) [inline docs/topics/kotlin-osgi.md:97]: No idea how to replace it, please help.

Comment (AlejandraPedroza) [reply docs/topics/kotlin-osgi.md:97]: Here's an option of a URL. This points to a release note explaining what are OSGi split package: https://docs.osgi.org/specification/osgi.core/7.0.0/framework.module.html#d0e5999

File: docs/topics/ksp/ksp-why-ksp.md (hunk 1 above)
Comment (AlejandraPedroza) [reply docs/topics/ksp/ksp-why-ksp.md:61]: I found this one: https://github.com/tachiyomiorg

**Review by AlejandraPedroza — CHANGES_REQUESTED** [review]
Thanks for fixing these links 👍 
 I added suggestions for the ones without a replacement.

**Comment by nikpachoo** [comment]
@AlejandraPedroza, thank you for your help. The PR is ready for review.

**Review by krutilov — APPROVED** [review]

**Review by AlejandraPedroza — APPROVED** [review]
LGTM 🚀  Thanks for doing this (:

**Pull request merged**